<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.cedarsoft.test.utils.CatchAllExceptionsExtensionTest" tests="4" skipped="4" failures="0" errors="0" timestamp="2021-05-07T02:08:11" hostname="runner-kggansxr-project-47-concurrent-0" time="0.032">
  <properties/>
  <testcase name="ignoredAssertion()" classname="com.cedarsoft.test.utils.CatchAllExceptionsExtensionTest" time="0.006">
    <skipped/>
  </testcase>
  <testcase name="successfulTest()" classname="com.cedarsoft.test.utils.CatchAllExceptionsExtensionTest" time="0.002">
    <skipped/>
  </testcase>
  <testcase name="testFailing()" classname="com.cedarsoft.test.utils.CatchAllExceptionsExtensionTest" time="0.006">
    <skipped/>
  </testcase>
  <testcase name="testIt()" classname="com.cedarsoft.test.utils.CatchAllExceptionsExtensionTest" time="0.002">
    <skipped/>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
